sdk/storage/inc/common/constant.hpp

```diff
@@ -0,0 +1,17 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+namespace Azure { namespace Storage {
+
+  constexpr static const char* k_ConnectionStringTagAccountName = "AccountName";
```

Comment:
Is it a `Tag`?

Reply:
I think I'm OK with both tag/key.

Reply:
These should also be in `Details` namespace, right?

Reply:
Should they start with `g_` prefix or `c_` prefix?

Reply:
Yes, I'll change it in the next iteration.

Reply:
should we change it to `g_` or `c_`?

Reply:
According to our previous discussion, this is going to put in details namespace so I guess it will be prefixed with `c_`.